Hello Bluesky.ipynb

```diff
@@ -88,12 +88,22 @@
     "### Executing a `count` plan with various parameters"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the example below, the Bluesky run engine is the interpreter of experiment plans and `count` is an experiment plan used here to acquire one reading from a point detector."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
+    "from bluesky import RunEngine\n",
+    "RE = RunEngine({})\n",
```

Comment:
This would shadow the `RE` added to the namespace by `%run beamline_configuration.py` above, which had some useful subscriptions on it.

Discussion with @bruceravel led me to the conclusion that the best way forward here is to refactor all our notebooks to move away from

```py
%run beamline_configuration.py  # magically add pre-configured RE, catalog, bec, and more to the namespace
```

toward

```py
from bluesky import RunEngine
RE = RunEngine()
from bluesky_tutorial_utils import setup_data_saving
catalog = setup_data_saving(RE)
bec = setup_live_feedback(RE)
```

This strikes a balance between showing clearly the steps of what has been done and being succinct so as not to detract from the main narrative of any given notebook.